### PR TITLE
ci: change speculos transport with http one

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Speculos are using mostly websocket to communicate, but it's seems this way of exchanging info is deprecated, so we change it to rest api

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts [hunk 1]
```diff
@@ -51,8 +67,10 @@ export default class SpeculosHttpTransport extends Transport {
         .then((response) => {
           response.data.on("data", (chunk) => {
             log("speculos-event", chunk.toString());
-            // XXX: we could process display events here
-            // client side automation or UI tests/checks
+            const split = chunk.toString().replace("data: ", "");
+            // FIXME: ugly patch because speculos return us a `data: `
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts:71]: Comment below code again? :smile:

**Review by haammar-ledger — APPROVED** [review]

File: libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts:71]: I don't know why I like comment below so much ...

Comment (gre) [reply libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts:71]: to me it's not an ugly thing, i think it's the spec of Event Stream that do this 🤔 
https://html.spec.whatwg.org/multipage/server-sent-events.html

so i think you can remove the comment.

File: libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts [hunk 2]
```diff
@@ -80,7 +98,7 @@ export default class SpeculosHttpTransport extends Transport {
       const action = { action: "press-and-release" };
       log("speculos-button", "press-and-release", but);
       this.instance
-        .post(`/button/${but}`, action)
+        .post(`/button/${this.buttonTable[but]}`, action)
```
Comment (gre) [inline libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts:101]: to be backward compat, you may need to do `this.buttonTable[but] || but`

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**